src/org/labkey/test/tests/assay/AssayTransformImportUpdateTest.java

```
@@ -224,6 +229,9 @@ public void testEnableTransformForUpdate() throws Exception
     @Test
     public void testCancelAsyncAssayTransformJob() throws Exception
     {
+        Assume.assumeTrue("Issue 53240: User cannot cancel pipeline job on SQL",
+                WebTestHelper.getDatabaseType() != WebTestHelper.DatabaseType.MicrosoftSQLServer);
+
```

I don't understand the value of this code. If this test fails in MSSQL you are just trading one exception for another. This adds better logging but not much else. It will still add to the failure noise in a MSSQL run.

I would suggest, either:
1. Remove this and let the test fail in MSSQL and add the comment in TeamCity and mute the failure.
2. Use this as a condition to skip part, or all(?), of the test if run on MSSQL.
3. Put it in its own test class that implements PostgressOnly.
4. Something else?

Options 2 & 3 aren't great options. They would require updates to the test when the issue is fixed. Not terrible, just something that needs to be tracked in the issue.

Option 1, muting in TeamCity, has the advantage of no update needed to the test code and it would start passing once the issue is fixed. On the down side if you don't scope the mute correctly you could hide a failure inPostgres.